packages/story-editor/src/app/media/uploadQueue/index.ts

```diff
@@ -476,65 +529,69 @@ function useMediaUploadQueue() {
       /**
        * Uploads a single pending item.
        *
-       * @param {Object} item Queue item.
-       * @param {File} item.file File object.
-       * @param {Object} item.additionalData Additional Data object.
+       * @param item Queue item.
+       * @param item.file File object.
+       * @param item.additionalData Additional Data object.
        */
-      async (item) => {
-        const {
-          id,
-          file,
-          resource,
-          isAnimatedGif,
-          state: itemState,
-          muteVideo,
-          cropVideo,
-          trimData,
-        } = item;
-        if (ITEM_STATUS.PENDING !== itemState) {
-          return;
-        }
+      (item: QueueItem) => {
+        void (async () => {
```

Comment:
I'm still not loving the IIFE, but here it allows `await` inside the function body, but that's a bit lazy IMO. IIFE's are definitely code smell. It's a bit more work to refactor it out though.